### PR TITLE
Push `latest` image tag

### PR DESCRIPTION
**Pull request by giorgiosironi**
We are pushing consistently images through this script. 

The script only has a specific "master branch" case, since the builds of pull requests happen from forks and are not allowed to use credentials to push any Docker image.

We don't have a safety net of integration tests outside of the project's own Travis CI build, so the `after_success` stage this scripts runs in is reached after all available automated tests have been run.

### CODE REVIEW

File: travis/push-image.sh [hunk 1]
```diff
@@ -21,6 +21,9 @@ if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_PULL_REQUEST" = "false" ]]; then
     # tag temporarily as liberoadmin due to lack of `libero/` availability
```
Comment (thewilkybarkid) [inline travis/push-image.sh:21]: We should contact them again.

**Review by thewilkybarkid — APPROVED** [review]
> We don't have a safety net of integration tests outside of the project's own Travis CI build, so the `after_success` stage this scripts runs in is reached after all available automated tests have been run.

That's fine, given Libero will be using tagged releases.

File: travis/push-image.sh (hunk 1 above)
Comment (giorgiosironi) [reply travis/push-image.sh:21]: We have a minimal paid plan on `elifesciences`, maybe that will help?

Comment (thewilkybarkid) [reply travis/push-image.sh:21]: 💵

**Comment by giorgiosironi** [comment]
Added `Git` to `search` in https://app.dependabot.com/accounts/libero

**Pull request merged**